bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -263,8 +271,9 @@ protected void initModel() {
             }
 
             if (useAssetDelivery) {
-                src = AssetDeliveryHelper.getSrc(assetDelivery, resource, imageName, extension, smartSizes,
-                    jpegQuality);
+                src = AssetDeliveryHelper.getSrc(assetDelivery, resource, imageName, extension,
+                        ArrayUtils.isNotEmpty(smartSizes) && smartSizes.length == 1 ? smartSizes[0] : null,
+                        ArrayUtils.isNotEmpty(smartSizes) ? jpegQuality : null);
```

Comment:
@vladbailescu `jpegQuality` is added only for `smartSizes.length == 1` for non asset delivery scenario - [here](https://github.com/adobe/aem-core-wcm-components/pull/2217/files#diff-cec7c952531c63b71fe29597d3e81dbdfc0549f771153046c5a304a45e9ee117R281). Don't we want to follow that here as well ?